test/integration/policy_info_metric_test.go

```diff
@@ -78,8 +82,16 @@ var _ = Describe("Test policy_governance_info metric", func() {
 		common.OcHub("apply", "-f", compliantPolicyYaml, "-n", userNamespace)
 		// Don't need to check compliance - just need to guarantee there is a policy in the cluster
 
-		token, err := common.OcHub("whoami", "-t")
+		//get token from SA
+		_, err := common.OcHub("create", "serviceaccount", saName, "-n", saNamespace)
```

Comment:
We probably need to make sure we're cleaning up this ServiceAccount in "Cleans up", also.